streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -475,8 +473,6 @@ public void shouldNotCommitBeforeTheCommitInterval() {
         mockTime.sleep(commitInterval - 10L);
         thread.setNow(mockTime.milliseconds());
         thread.maybeCommit();
-
-        verify(taskManager);
```

Comment:
The TaskManager is created in the mockTastManager* methods. I have taken the approach of changing all methods in there including the ones with a return value void to when() statements. Mockito automatically verifies that setup statements have been called so these verifications are no longer necessary.